jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -650,17 +650,32 @@ ecma_builtin_try_to_instantiate_property (ecma_object_t *object_p, /**< object *
     {
       /*
        * Lazy instantiation of 'length' property
-       *
-       * Note:
-       *      We don't need to mark that the property was already lazy instantiated,
-       *      as it is non-configurable and so can't be deleted
        */
 
       ecma_property_t *len_prop_p;
+#if ENABLED (JERRY_ES2015)
+      ecma_extended_object_t *ext_func_p = (ecma_extended_object_t *) object_p;
+      uint32_t *bitset_p = ext_func_p->u.built_in.instantiated_bitset;
+      if (*bitset_p & (1u << 0))
```

Comment:
This is kind of confusing for me: how do we know that the 0th bit represents the length?

Reply:
`instantiated_bitset` member is defined but never used for builtin functions.
(It is used to mark each instantiated property of builtin objects such as `Array`, but currently builtin functions also share this structure although never use it)
So, I arbitrarily use the last bit as instantication marker for 'length' property.
I also have a plan to revise `instantiated_bitset` for builtin functions in following patches.